pkgs/development/libraries/libwebp/default.nix

```
@@ -50,6 +55,7 @@ stdenv.mkDerivation rec {
     (mkFlag libwebpdecoderSupport "libwebpdecoder")
   ];
 
+  nativeBuildInputs = [ autoconf automake libtool ];
```

```suggestion
  nativeBuildInputs = [ autoreconfHook libtool ];
```

Then you can remove the preConfigure hook.